src/srctools/_fgd_helpers.py

```diff
@@ -690,11 +695,30 @@ def export(self) -> List[str]:
         """Produce the arguments for iconsprite()."""
         return [self.theta, self.phi, self.color]
 
-
+@attr.define
 class HelperRope(Helper):
     """Specialized helper for displaying move_rope and keyframe_rope."""
+
     TYPE: ClassVar[HelperTypes] = HelperTypes.ENT_ROPE
 
+    name_kv: str
```

Comment:
This needs to be Optional, to indicate it can also be set to None.
```suggestion
    name_kv: Optional[str]
```